app-editors/scite/scite-4.1.2.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic gnome2-utils toolchain-funcs xdg-utils
+
+MY_PV=${PV//./}
+DESCRIPTION="A very powerful, highly configurable, small editor with syntax coloring."
+HOMEPAGE="http://www.scintilla.org/SciTE.html"
+SRC_URI="http://www.scintilla.org/${PN}${MY_PV}.tgz"
```

Comment:
https://, also please append `-> ${P}.tgz` to make the distfile match the ebuild.